tests/pages/services/ServiceTable-cy.js

```diff
@@ -234,9 +234,9 @@ describe("Service Table", function() {
         .should("to.have.length", 1);
 
       cy
-        .get(".modal pre")
+        .get(".modal-body")
```

Comment:
I don't think removing one will create inconsistency because otherwise we could apply the same argument to the other tests that don't have duplicated.